### DISCUSSION TITLE
Options to revert New Popup to Old Popup Features

### DISCUSSION THREAD

**Discussion by pac-71** [coddingtonbear/obsidian-web#140] [Ideas]
This one is a bit personal but I prefer the look and feel of the old popup and it would be a good way to revet back to the original look and feel. In particular.

- Toggle same as browser theme setting plus an option to force dark or light theme.
- Add a toggle to clip details so the templates are automatically crystalized (as they were previously). This seriously adds details that are not required by people who infrequently change template details.
- The new positioning of the popup requires excessive movement from the Obsidian Web icon to the popup 
New Position v3..0.0
![ObsidianWebPosition 3 0 0](https://github.com/coddingtonbear/obsidian-web/assets/29050734/30136992-a982-4a4b-8e03-8bb34b786c99)
Old 2.2.4 Position
![ObsidianWebPosition 2 2 4](https://github.com/coddingtonbear/obsidian-web/assets/29050734/7402c7e3-961a-46e7-94e3-ca57655d6af3)

**Comment by coddingtonbear** [comment]
These are good thoughts, but I'm sorry to say that I'm not likely to implement them -- the more toggles there are, the more complexity there is to manage going forward over time, and complexity is the death of projects like this.

That said -- you did mention something interesting that I hadn't considered -- it seems like you think you need to crystalize the template for it to be applied.  So: let me be clear, you *do not* need to crystalize the template unless you want to change the output of the template before sending it to Obsidian.  That should maybe be clearer in the UI.  

Something else you might want to keep in mind is that you no longer need to ever click the browser extension button anymore now that there's a keyboard shortcut.  In fact: you can navigate through the whole thing without ever touching your mouse.
